packages/x-data-grid-premium/src/hooks/features/promptControl/types.ts

```diff
@@ -0,0 +1,29 @@
+export type Sort = {
+  column: string;
+  direction: 'asc' | 'desc';
+};
+
+export type Grouping = {
+  column: string;
+};
+
+export type Filter = {
+  operator: string;
+  value: string | number | boolean | string[] | number[];
+  column: string;
+};
+
+export type AggregationFunction = 'avg' | 'sum' | 'min' | 'max' | 'size';
+export type Aggregation = {
+  [column: string]: AggregationFunction;
+};
+
+export type PromptResponse = {
+  select: number;
+  filters: Filter[];
```

Comment:
I'm wondering if we keep it aligned with `GridFilterModel` to keep the possibilities such as the following open:
```
// Search all the data for a text containing with "John" or "Abraham"
filterModel = {
  items: [],
  quickFilterLogicOperator: GridLogicOperator.Or,
  quickFilterValues: ["John", "Abraham"],
}
```
```suggestion
  filterModel: GridFilterModel;
```

Wdyt?